pkg/cmd/rollbackcopy/fileioutils.go

```diff
@@ -0,0 +1,152 @@
+package rollbackcopy
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"k8s.io/klog"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+)
+
+func fileExists(filename string) bool {
+	info, err := os.Stat(filename)
+	if os.IsNotExist(err) {
+		return false
+	}
+	return !info.IsDir()
+}
+
+func dirExists(dirname string) bool {
+	info, err := os.Stat(dirname)
+	if os.IsNotExist(err) {
+		return false
+	}
+	return info.IsDir()
```

Comment:
Unsafe deference of `info` when `err` is not nil but and not a NotExist error. Need to return any unexpected error immediately.